packages/utils/fluid-main/src/MergeTree.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export * from  "@fluidframework/merge-tree";
```

Comment:
I don't think merge tree should be directly access by the developer? @anthony-murphy

Reply:
Looks like RichTextAdapter of draft-js is using merge-tree. If this package won't commonly be used, we can certainly omit it.

https://github.com/microsoft/FluidExamples/blob/95768cfe1cc045a400d19ad3b74da574f7133665/draft-js/src/utils/RichTextAdapter.ts#L13

Reply:
@anthony-murphy should chime in here.   I believe the intent is that MergeTree is internal implementation detail and sequence is the user facing API.   If there are type that MergeTree needs to be exposed, I think we should re-export that thru sequence as well?

Reply:
Sounds good. We can always add it back in later.